src/main/scala/com/databricks/spark/xml/XmlOptions.scala

```diff
@@ -62,7 +62,7 @@ private[xml] class XmlOptions(
   val permissive = ParseModes.isPermissiveMode(parseMode)
 
   require(rowTag.nonEmpty, "'rowTag' option should not be empty string.")
-  require(attributePrefix.nonEmpty, "'attributePrefix' option should not be empty string.")
+  logger.warn("'attributePrefix' option should not be empty string.")
```

Comment:
I think this still needs to be conditional, else it will always fire the warning?

Reply:
@RJKeevil one xml = one warning
For my xml i need empty attributePrefix, because all my values in attributes

Reply:
My task it's use fias (All addresses of Russia) xml in spark.
I think many Russains will collide, in this problem.

Reply:
I understand, but the way the code is written it will fire the empty attribute prefix warning, whether it is empty or not? I have a prefix of _ defined and I get that warning in the logs

Reply:
Oy, i understand, i will fix it.